keyboards/handwired/daishi/rules.mk

```
@@ -0,0 +1,66 @@
+# MCU name
+MCU = atmega32u4
```

You have the wrong MCU set here.

```suggestion
MCU = at90usb1286
```